Anlab.Core/Domain/PaymentEvent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Anlab.Core.Domain
+{
+    public class PaymentEvent
+    {
+        public PaymentEvent()
+        {
+            OccuredAt = DateTime.UtcNow;
+        }
+        [Key]
+        public string Transaction_Id { get; set; }
+
+        /// <summary>
+        /// Unique merchant-generated order reference or tracking number for each transaction.
+        /// (Order Id)
+        /// </summary>
+        public int Req_Reference_Number { get; set; }

```

Comment:
@jSylvestre in addition to req request number, which is the orderID, we should really have a FK link to the Order itself, you know like `public Order Order {get;set;}`

Reply:
@srkirkland This is info back from CyberSource, I wanted to capture everything then return back even bad stuff, so I didn't want FKs. Let me know if you still want that.